content/new-web-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+---
+title: New website infrastructure.
+description: >
+ A troubleshooting triage to help you self-diagnose a variety of issues.
+keywords:
+ - support
+ - software
+ - environment
+ - warranty
+ - self-help
+ - tech-support
+ - troubleshooting
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: true
+section: getting-help
+tableOfContents: true
+---
+
+Hello!
+
+You have landed on a place holder page as our web site has had some changes as of Novemember 1st, 2024. Links in support emails that land here are no longer supported by the new site.
+
+We do want to help you if you do have questions, concerns or on going issues! If you are logged into your System76.com account, the following link will allow you to create a support ticket now:
+
+[Open New Support Ticket](https://system76.com/my-account/support-tickets/new)
+
+## Suggested self help fisrt steps
+
+### 1. Do you have a current backup of your files?
+
+If your issue is time-sensitive, reinstalling or [refreshing](/articles/pop-recovery#refresh-install) the operating system may provide the fastest resolution. Be sure to back up all important data before performing any tests or contacting Support. Re-imaging is much easier to do if we know that your important files are safe.
+
+**Note**: This guide may cover diagnostics and testing that you've already performed. Skip those sections as needed, but be sure to provide that information when submitting your ticket.
+
+We have a help article to guide you through backing up your system [here.](/articles/backup-files)
+
+### 2. Do you have a Pop!\_OS or Ubuntu Live USB?
+
+A Live USB can be used as a rescue disk. This will provide us an outside OS environment to access your installed system, potentially rescue files that have not been backed up, or quickly reinstall the OS if necessary.
+
+We have an article to guide you through creating [Live USBs here.](/articles/live-disk)
+
+## Local repair and upgrades
+
+We offer techincal documenation (tech-docs such as service manuals) in this [article](/articles/guides).
+
+## Common Issue Sources
+
+### Software
+
+See this article for troubleshooting software issues [here.](/articles/software-triage)
```

Comment:
If you're going to say "see this article," it would be better to end with a colon and then a URL link, rather than a "here" hyperlink. You could also hyperlink the words "this article" instead, or reword this a bit and hyperlink "troubleshooting software issues." ("Here" is always a vague link to click, and to direct people to.)